docs/docs/guides/cli/how-to-auto-generate-schema-files.mdx

```diff
@@ -61,12 +62,13 @@ lightdash generate -s payments+ +customers tag:prod # mix and match
 
 For example, if I have a field called `product_category` that's used a bunch of times throughout my project, I can create a doc block for this field:
 
-```markdown
+````markdown
```

Comment:
🤔

Reply:
Ok, I understand why, because there's a nested code block

Reply:
Ended up tweaking this page, not sure if it was intentional or not but part of the guide text was inside the code block - didn't look horrible, but it makes more sense to break things out into more than one code block.